singer-io/tap-adwords
# Fixes broken links in readme (SPEC & SCHEMA)

Both links yielded 404 because the original documentation changed. The SPEC was moved into a sub folder, but the schema one might not be directly equivalent. 
In the second case, I've selected the part that best represents the intentions of this section

# Description of change
Two URLs in the `README.md` were changed because the links were broken

# Manual QA steps
 - click the new URLs
 - check if the docs are the ones intended
 
# Risks
 - none
 
# Rollback steps
 - revert this branch
